constrain/Classes/BulkOperations.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  BulkOperations.swift
+//  constrain
+//
+//  Created by New Greg on 3/25/20.
+//
+
+import Foundation
+
+public typealias ConstraintSet = [NSLayoutConstraint]
```

Comment:
Might be a misleading name, since Set implies that only one of each kind is allowed?

Reply:
oh yeah, maybe just `Constraints`, or keep it explicitly an array